src/test/resources/loose-war-with-loose-jar/ejb-ejb/src/main/java/wasdev/ejb/ejb/SampleStatelessBean.java

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
```

Comment:
This copyright seems out of place. In general we don't have copyrights in test case files.